lib/core/utils.js

```diff
@@ -361,11 +361,20 @@ RESUtils.thingsContainer = function(body) {
 
 	return container;
 };
-RESUtils.things = function(container) {
+RESUtils.$things = function(container) {
 	container = container || RESUtils.thingsContainer();
 	return $(container).find(RESUtils.thing.prototype.thingSelector);
 };
+RESUtils.things = function(container) {
```

Comment:
`.things` returns an array of thing objects (the view wrapper for things elements with accessors for various properties/elements); `.$things` returns a jquery set of elements which can be wrapped in `.things`.  Does that make sense or should we pick some different names?